modules/simple_bucket/outputs.tf

```diff
@@ -33,3 +33,11 @@ output "internal_kms_configuration" {
   description = "The intenal KMS Resource."
   value       = var.internal_encryption_config.create_encryption_key ? module.encryption_key[0] : null
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//storage.googleapis.com/${element(split("/b/", google_storage_bucket.bucket.self_link), 1)}"
+    service_id  = substr(format("%s-%s", google_storage_bucket.bucket.name, md5(var.project_id)), 0, 63)
```

Comment:
nit: This might redundant as bucket names are unique. we can simply keep.
`service_id = google_storage_bucket.bucket.name`

Reply:
Make sense, I still kept the substring part in case the bucket name size exceeds 64.